dev/com.ibm.ws.jsfContainer_fat_2.3/fat/src/com/ibm/ws/jsf/container/fat/tests/ClassloadingTest.java

```diff
@@ -112,17 +138,21 @@ private void runTest() throws Exception {
         HttpUtils.findStringInReadyUrl(server, '/' + JSF_APP + "/TestBean.jsf",
                                        "CDI Bean value:",
                                        ":CDIBean::PostConstructCalled:");
+      if(!isEE10){
         HttpUtils.findStringInReadyUrl(server, '/' + JSF_APP + "/TestBean.jsf",
                                        "JSF Bean value:",
                                        ":JSFBean::PostConstructCalled:");
+      }
 
         // Verify that basic JSF works in an EAR
         HttpUtils.findStringInReadyUrl(server, '/' + JSF_EAR_APP + "/TestBean.jsf",
                                        "CDI Bean value:",
                                        ":CDIBean::PostConstructCalled:");
-        HttpUtils.findStringInReadyUrl(server, '/' + JSF_EAR_APP + "/TestBean.jsf",
-                                       "JSF Bean value:",
-                                       ":JSFBean::PostConstructCalled:");
+        if(!isEE10){
+          HttpUtils.findStringInReadyUrl(server, '/' + JSF_EAR_APP + "/TestBean.jsf",
```

Comment:
This is the ManagedBean test?

Reply:
It is. Not sure what you mean since `!isEE10` allows it for run for everything but EE10?

Reply:
I was just making sure I understood why we were skipping this part of the test for EE10. We're all set thanks for the clarification.